ambry-protocol/src/test/java/com.github.ambry.protocol/RequestResponseTest.java

```diff
@@ -494,8 +494,10 @@ public void catchupStatusAdminRequestTest() throws IOException {
     String clientId = "client";
     // request
     long acceptableLag = Utils.getRandomLong(TestUtils.RANDOM, 10000);
+    short numCaughtUpPerPartition = (short) TestUtils.RANDOM.nextInt(Short.MAX_VALUE);
```

Comment:
We have a method to generate randomShort. Utils.getRandomShort(Random random)

Reply:
Switched to use method.